nxcomp/Misc.cpp

```diff
@@ -358,6 +358,13 @@ tolerancechecks=s\n\
                ory compared to the other programs, to make easier\n\
                for the user to execute the program from the shell.\n\
 \n\
+  NX_SLAVE_CMD The full path to the slave channel handler. When the\n\
+               slave channel is enabled, the agent will listen on a\n\
+               port and forward the connection to the NX_SLAVE_CMD\n\
```

Comment:
And yes, the command is a multiplexer. In our particular usage, it's an HTTP server. So it works like this:

GET /ping
Pong!

POST /tcp/connect/4000
(slave handler execs socat, now the connection starts talking to port 4000)

I made a little test script that works on the same principle, just without HTTP and without using our libraries. I will add it to testscripts.